### DISCUSSION TITLE
Cannot create a Univer Sheet instance without using presets.

### DISCUSSION THREAD

**Discussion by tranphat180603** [dream-num/univer#4390] [Q&A | 常见问题]
Hi, I have tried created Univer sheet instance using the preset method. And it works really well. Here's the images of the code and the sheet:

<img width="284" alt="1" src="https://github.com/user-attachments/assets/57227c47-57f1-48bd-a620-003e11365c9c" />
<img width="959" alt="image" src="https://github.com/user-attachments/assets/6fa76d1a-24ce-4add-8034-08dd6e1738b3" />

However, when I tried to use the piecemeal installation approach and follow exactly step-by-step to the instruction on this URL: https://docs.univer.ai/en-US/guides/sheets/getting-started/installation
It works without any problem but it does not show any cells unlike the preset method. I tried many different approaches and still not yet find a solution, please help!

<img width="853" alt="image" src="https://github.com/user-attachments/assets/61b54c54-1dde-41ba-a61b-8d4324bc57c7" />
<img width="958" alt="image" src="https://github.com/user-attachments/assets/7ed27282-ca2c-4235-8bd3-0e478f0b4a0d" />

**Comment by jikkai** [comment]
You just forgot to set a height for the container in the CSS :)

**Reply by tranphat180603** [reply]
<img width="259" alt="image" src="https://github.com/user-attachments/assets/198883be-0d8f-42df-93b0-442ee169944e" />

you mean this? If yes, it still does not work tho

**Reply by jikkai** [reply]
https://stackoverflow.com/questions/5657964/css-why-doesn-t-percentage-height-work

**Reply by tranphat180603** [reply]
<img width="416" alt="image" src="https://github.com/user-attachments/assets/a07d2e92-be43-4ffa-b2b3-d120de68c7ee" />

I changed it too but does not work either. I doubt that the problem comes from setting the height. Although I really don't understand why. I cloned some of the repo sheets-react-demo and compile and run the whole thing with no problem so that definitely nothing wrong with the current settings of my computer. 

I just followed the whole instruction of installation and everything on your documentation website but apparently, some of the function seems deprecated and it's just buggy with import problem, etc. 

Note: I use the latest version 0.5.2

**Reply by jikkai** [reply]
If you still think it's an issue with Univer, please provide the minimum reproduction.
